components/src/core/score-card/score-card.component.scss

```diff
@@ -38,34 +33,37 @@
         pointer-events: none;
     }
     .pxb-score-card-header-wrapper {
-        padding: 16px;
-        padding-bottom: 0;
+        padding: 1rem 16px 0 16px;
         display: flex;
         justify-content: space-between;
     }
     .pxb-score-card-action-items-wrapper {
         text-overflow: unset;
         overflow: unset;
-        margin-right: -12px;
+        margin-right: -0.75rem;
         > * {
             cursor: pointer;
             margin-right: 12px;
             text-align: center;
+            @include scale-icon-content();
```

Comment:
For the very right icon, the right margin should be 16px.
![image](https://user-images.githubusercontent.com/8997218/109887874-8ffecb00-7c50-11eb-887e-aab9e79fae87.png)

Reply:
Updated this.